src/Gateway/OneyGatewayFactory.php

```diff
@@ -9,4 +9,13 @@ final class OneyGatewayFactory extends AbstractGatewayFactory
     public const FACTORY_NAME = 'payplug_oney';
 
     public const FACTORY_TITLE = 'Oney by PayPlug';
+
+    public const AUTHORIZED_CURRENCIES = [
+        'EUR' => [
+            'min_amount' => 10000,
+            'max_amount' => 300000,
```

Comment:
Are we sure we have to use this consts ? These information can be retrieve through getAccount endpoint.

Reply:
Well, I implemented this check from getAccount in my next PR. We'll change it later :wink:

Reply:
Petit warning ici, on risque à terme d'avoir des montants min et max différents par marchand. Je vois que vous en avez deja conscience mais je prefere quand même le préciser. Bien entendu, on est encore un peu loin d'avoir ce genre de problème sur Sylius.